src/lt/plugins/clojure.cljs

```diff
@@ -517,18 +549,40 @@
 ;; watches
 ;;****************************************************
 
+;; For more information on watches check
+;;
+;; Original anouncement: https://groups.google.com/forum/#!msg/light-table-discussion/lyFzPGI2XMs/ec8T1OUPvMsJ
+;;
+;; Blog posts: http://scattered-thoughts.net/blog/2014/01/27/were-not-even-trying/?utm_source=dlvr.it&utm_medium=twitter
+;;             https://medium.com/@zindlerb/guide-to-light-table-watches-fad560f698d3#.oqwq991sx
+;;
+;; Rolex watches plugin: https://groups.google.com/forum/#!topic/light-table-discussion/NQWGC0vVHMY
+
 (behavior ::cljs-watch-src
+          :desc "Clojurescript: Wraps the watched source code"
+          :doc  "Wraps watched code with a call to `js/lttools.watch` to catch its result and
```

Comment:
Documentation shouldn't be specifying implementation. Please replace`js/ltools.watch` with a description of what it does or recommend removing :doc